src/main/java/net/consensys/orion/config/Config.java

```diff
@@ -659,6 +683,73 @@ private static Schema configSchema() {
         "TLS known servers for the client. This contains the fingerprints of public keys of other nodes that this node has encountered for the ca-or-tofu, tofu and whitelist trust modes.",
         null);
 
+    schemaBuilder.addString(
+        "clientconnectiontls",
+        "off",
+        "TLS status. Options:\n"
+            + "\n"
+            + "   - strict: All connections to the client connection of this node must use TLS with client authentication\n"
+            + "       See the documentation 'clientconnectiontlsservertrust'\n"
+            + "   - off: Mutually authenticated TLS is not used for in- and outbound\n"
+            + "       connections, although unauthenticated connections to HTTPS hosts are still possible. This\n"
+            + "       should only be used if another transport security mechanism like WireGuard is in place.",
+        PropertyValidator.anyOfIgnoreCase("off", "strict"));
+
+    schemaBuilder.addString(
+        "clientconnectiontlsservertrust",
+        "tofu",
+        "TLS trust mode for the internal server endpoint. This decides who's allowed to connect to it. Options:\n"
```

Comment:
break options to new line to make it easier to see when scanning down the page.

Reply:
not sure I understand - example?

Reply:
> ...connect to it. Options:\n
vs
> connect to it.\nOptions:\n

So that in the usage, its

TLS trust mode for the internal server endpoint. This decides who's allowed to connect to it.
Options:
 - blah...

Reply:
Ahhhh - going to be difficult and say taht Given this was cut-and-paste, don't want to change all instances.